### ISSUE TITLE
templatefile and templateLocation parameters missing/incorrect in doco

### ISSUE THREAD

**Issue by ShadowXVII** [rfennell/AzurePipelines#1056]
### Azure DevOps Extensions

Generate Release Notes (Node Cross Platform)

### Platform

Azure DevOps Services

### Describe the bug

```bash
Seems the table of parameters for the task is out of date. For example, templateFile and templateLocation doesn't work the same as documented in the table.

Is there a more up to date list or documentation?
```

### Repo Steps

```bash
Using https://marketplace.visualstudio.com/items?itemName=richardfennellBM.BM-VSTS-XplatGenerateReleaseNotes&ssr=false#overview as documentation.
```

**Comment by ShadowXVII** [comment]
Oh, I just found https://github.com/rfennell/AzurePipelines/wiki/GenerateReleaseNotes---Node-based-Cross-Platform-Task-YAML

That's a lot more useful -- perhaps the main overview doco on the VS page can link to the relevant GitHub articles if they're the "source of truth" doco wise.

**Comment by rfennell** [comment]
Which document are you referring to as the 'main overview', I am a little confused?

I ask because the `readme.md` in each extension folder is used as the marketplace entry page, and is also copied to the [WIKI](https://github.com/rfennell/AzurePipelines/wiki), this is supplemented by autogenerated (from the manifests) YAML usage documentation.

**Comment by ShadowXVII** [comment]
Yes, the marketplace entry -- which would therefore come from this [readme.md](https://github.com/rfennell/AzurePipelines/tree/main/Extensions/XplatGenerateReleaseNotes#task-parameters)

The overview should perhaps have information about the YAML task vs the Pipeline task and links to the relevant wiki articles? I found it tricky to get started because the parameters listed on that task table didn't match the YAML ones.

**Comment by rfennell** [comment]
For the next release, I will edit the readme so that it at least references the YAML page in the wiki. Also, it seems I need to review the readme to check all the listed parameters are correct.

Thanks for the input
